jwql/website/apps/jwql/views.py

```diff
@@ -402,16 +405,77 @@ def dashboard(request):
         Outgoing response sent to the webpage
     """
 
+    from bokeh.layouts import column, layout
+    from bokeh.embed import components
+
     template = 'dashboard.html'
-    output_dir = get_config()['outputs']
-    dashboard_components, dashboard_html = get_dashboard_components()
 
-    context = {'inst': '',
-               'outputs': output_dir,
-               'filesystem_html': os.path.join(output_dir, 'monitor_filesystem',
-                                               'filesystem_monitor.html'),
-               'dashboard_components': dashboard_components,
-               'dashboard_html': dashboard_html}
+    db = get_dashboard_components(request)
+    pie_graph = db.dashboard_instrument_pie_chart()
+    files_graph = db.dashboard_files_per_day()
+    filetype_bar = db.dashboard_filetype_bar_chart()
+    table_columns, table_values = db.dashboard_monitor_tracking()
+    grating_plot = db.dashboard_exposure_count_by_filter()
+    anomaly_plot = db.dashboard_anomaly_per_instrument()
+
+    p = layout([
+        [files_graph],[pie_graph, filetype_bar],[grating_plot, anomaly_plot]
+        ],sizing_mode='stretch_width') 
+    script, div = components(p)
+
+    time_deltas = ['All Time', '1 Day', '1 Week', '1 Month', '1 Year']
+
+    context =  {'inst': '',
+               'script': script,
+               'div': div,
+               'table_columns': table_columns,
+               'table_rows': table_values,
+               'time_deltas':time_deltas}
+
+    return render(request, template, context)
+
+
+def daily_trending(request, date):
+    """Generate the daily dashbaord page
+
+    Parameters
+    ----------
+    request : HttpRequest object
+        Incoming request from the webpage
+
+    Returns
+    -------
+    HttpResponse object
+        Outgoing response sent to the webpage
+    """
+
+    from jwql.website.apps.jwql.bokeh_dashboard import generalDashboard
+    from bokeh.plotting import figure, output_file, show 
+    from bokeh.embed import components
+    from bokeh.layouts import column, layout
+    import numpy as np
```

Comment:
Could these imports be put at the top of the module instead of inside this function?